test/evergreen/run_format_configs.sh

```diff
@@ -26,11 +26,12 @@ failure=0
 for config in $(find ../../../test/format/failure_configs/ -name "CONFIG.*" | sort)
 do
 	echo -e "\nTesting CONFIG $config ...\n"
-	if (./t -c $config); then
+	if (./t -c $config -h WT_TEST); then
 		let "success++"
 	else
 		let "failure++"
-		[ -f RUNDIR/CONFIG ] && cat RUNDIR/CONFIG
+		[ -f WT_TEST/CONFIG ] && cat WT_TEST/CONFIG
+		mv WT_TEST WT_TEST_${config##*/}
```

Comment:
Small tweak: We can strip the parent folders from `config` using `basename`.

I'd change 
`mv WT_TEST WT_TEST_${config##*/}`
to either 
`mv WT_TEST WT_TEST_$(basename $config)`
or
```
config_name=$(basename $config)
mv WT_TEST WT_TEST_${config_name}    
```

Reply:
And we could probably make `WT_TEST` a variable :)